packages/react-charting/UnitTests/StackedBarChartUT.test.tsx

```diff
@@ -0,0 +1,186 @@
+import { StackedBarChartBase } from '../src/components/StackedBarChart/StackedBarChart.base';
+import { IChartDataPoint, IChartProps } from '../src/components/StackedBarChart';
+import { DefaultPalette } from '@fluentui/react';
+import { DarkTheme } from '@fluentui/theme-samples';
+const env = require('../config/tests');
+
+const runTest = env === 'TEST' ? describe : describe.skip;
+
+const points: IChartDataPoint[] = [
+  {
+    legend: 'first',
+    data: 3000000,
+    color: DefaultPalette.blue,
+    xAxisCalloutData: '2020/04/30',
+    yAxisCalloutData: '99%',
+  },
+  { legend: 'second', data: 1, color: DefaultPalette.green },
+];
+
+const areaLabelPoints: IChartDataPoint[] = [
+  {
+    legend: 'first',
+    data: 3000000,
+    color: DefaultPalette.blue,
+    xAxisCalloutData: '2020/04/30',
+  },
+  {
+    legend: 'first',
+    data: 3000000,
+    color: DefaultPalette.blue,
+    yAxisCalloutData: '50%',
+  },
+  {
+    data: 2000,
+    color: DefaultPalette.blue,
+  },
+];
+
+const data: IChartProps = {
+  chartTitle: 'Stacked Bar chart example',
+  chartData: points,
+};
+
+runTest('_createBarsAndLegends', () => {
+  test('Should return bars and legends which includes benchmark and target data', () => {
+    const instance = new StackedBarChartBase({
+      data: {},
+    });
+    expect(instance).toBeDefined();
+    const { palette } = DarkTheme;
+    const barHeight = 500;
+    const benchmarkData = points[0];
+    const targetData = points[0];
+
+    const barsAndLegends = instance._createBarsAndLegends(data, barHeight, palette, benchmarkData, targetData);
+    expect(barsAndLegends).toHaveLength(2);
+    const bars = barsAndLegends[0];
+    expect(bars).toHaveLength(2);
+    const legendsDataObject = barsAndLegends[1];
+    expect(legendsDataObject['props']).not.toBeNull();
+    const legends = legendsDataObject['props']['legends'];
+    expect(legends).toHaveLength(4);
+  });
+
+  test('Should return bars and legends which includes only benchmark data', () => {
+    const instance = new StackedBarChartBase({
+      data: {},
+    });
+    expect(instance).toBeDefined();
+    const { palette } = DarkTheme;
+    const barHeight = 500;
+    const benchmarkData = points[0];
+
+    const barsAndLegends = instance._createBarsAndLegends(data, barHeight, palette, benchmarkData);
+    expect(barsAndLegends).toHaveLength(2);
+    const bars = barsAndLegends[0];
+    expect(bars).toHaveLength(2);
+    const legendsDataObject = barsAndLegends[1];
+    expect(legendsDataObject['props']).not.toBeNull();
+    const legends = legendsDataObject['props']['legends'];
+    expect(legends).toHaveLength(3);
```

Comment:
where are you asserting for the benchmark data?

Reply:
We do have separate test for benchmark which is in component tests